### PR TITLE
Allow users for namespace creation.

### PR DESCRIPTION
**Pull request by jctanner**
https://issues.redhat.com/browse/AAH-2721

### CODE REVIEW

**Comment by rochacbruno** [comment]
This error on CI is related?

2023-09-25T15:29:27.7385689Z [32mpulp_1          |[0m pulp [fc674cd114a6440ea5dd045d7b6bb97c]: pulpcore.tasking.tasks:INFO: Task 018accf3-503d-75f9-8bb0-96b064e61e5a failed (AnsibleNamespaceMetadata() got unexpected keyword arguments: 'users')

**Comment by jctanner** [comment]
awaiting https://github.com/pulp/pulpcore/pull/4479

**Comment by jctanner** [comment]
also awaiting https://github.com/pulp/pulp_ansible/pull/1594

File: galaxy_ng/tests/integration/api/test_namespace_management.py [hunk 1]
```diff
@@ -56,3 +56,109 @@ def test_namespace_create_and_delete(ansible_config, api_version):
...
+        ]
+    }
+    resp = api_client(f'{api_prefix}/_ui/v1/my-namespaces/', args=payload, method='POST')
+
+    # should have the right results ...
+    assert resp['name'] == new_namespace
+    assert resp['groups'] == []
+    assert resp['users'] != []
+    assert username in [x['name'] for x in resp['users']]
+    assert username in [x['name'] for x in resp['users']]
```
Comment (jerabekjiri) [inline galaxy_ng/tests/integration/api/test_namespace_management.py:103]: duplicate assert

File: galaxy_ng/app/utils/pulp_rbac.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+def get_users_with_perms_attached_roles(
+    obj,
+    with_group_users=True,
+    only_with_perms_in=None,
+    include_domain_permissions=True,
+    include_model_permissions=True,
+    for_concrete_model=False,
+):
+    # DELETE ONCE https://github.com/pulp/pulpcore/pull/4479 IS RELEASED AND BUMPED IN GALAXY_NG
```
Comment (jerabekjiri) [inline galaxy_ng/app/utils/pulp_rbac.py:19]: Can be deleted now? looks like https://github.com/pulp/pulpcore/issues/4478 was released in 3.28.16.

**Review by bmclaughlin — DISMISSED** [review]

**Review by himdel — APPROVED** [review]

**Pull request merged**